src/ut/httpclient_test.cpp

```diff
@@ -216,3 +778,232 @@ TEST_F(HttpClientTest, SasDisplayName)
 
   mock_sas_collect_messages(false);
 }
+
+// Test creation and destruction of the basic http resolver
+TEST_F(HttpClientTest, BasicResolverTest)
```

Comment:
Why is this test here? Seems completely unrelated

Reply:
Ported across from the old httpconnection_test file, where it was if anything even more un-related. Agree, but don't know if i should really split out a whole new thing just for this.